coops-ui/src/views/Dashboard.vue

```diff
@@ -239,13 +239,15 @@ export default {
      * Searches the filings history for a 'paid' status.
      * Paid status indicates a filing that is paid but future effective.
      * Change the state of the UI when a filing is future effective.
-     * Currently only BCOMPS have future effective filings.
+     * Currently only BCOMPs change of addresses filing are future effective.
      *
      * @param filings The array of filings in history
      */
     checkPendingFilings (filings) {
       filings.forEach(filing => {
-        if (filing.status === FilingStatus.PAID) {
+        if (this.entityFilter(EntityTypes.BCORP) &&
+          filing.name === 'Address Change' &&
```

Comment:
I know these are assigned in FilingHistoryList.vue:loadData() -- ie, not your code -- but could you please go back and create an enum to both assign the name and compare it here?

Reply:
Great idea, i considered this as well. I'm guilty of not being pro-active on this one :) Lets do it!